lib/write_concern.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+/**
```

Comment:
This will be a publicly documented type, so please write thorough documentation. Specifically,  there should be documentation for the params passed into the constructor, the properties available on the type itself (since it's well known), and references to the mongodb docs re: write concerns (in a "see also" section maybe as part of the class-level docs).